pramen/core/src/main/scala/za/co/absa/pramen/core/databricks/Schema.scala

```diff
@@ -0,0 +1,19 @@
+package za.co.absa.pramen.core.databricks
+
+object Schema {
+  case class RunJobResponse(runId: String)
+
+  case class RunState(lifeCycleState: String)
+
+  case class RunStatusResponse(runId: String, runName: String, runPageUrl: String, state: RunState) {
+    def isPending: Boolean = {
+      state.lifeCycleState == "TERMINATED" || state.lifeCycleState == "INTERNAL_ERROR"
```

Comment:
This is a strange combination of the method name and implementation 😆

Maybe you meant 'isFinished' ?

Reply:
ah crap 😄 thanks for noticing. Don't know what I was thinking